tinyinst.cpp

```diff
@@ -850,6 +851,8 @@ void TinyInst::OnModuleInstrumented(ModuleInfo* module) {
       }
       if(address) {
         resolved_hooks[address] = hook;
+      } else {
+        FATAL("Could not resolve function %s in module %s", hook->GetFunctionName().c_str(), hook->GetModuleName().c_str());

```

Comment:
We don't want to FATAL() here because hook's module name could be "*" so it searches for the function in all modules.

Reply:
Ah that makes good sense. Do you foresee a better way to alert the user if a hook function is unable to be resolved? Or do you think it's not necessary?

Reply:
I suppose I could just change it to SAY()